### PR TITLE
#330: Adding meaningful messages for regex pattern errors

### PR DESCRIPTION
**Pull request by hani-ibrahim**
#330 

Not sure about the messages 😖

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -71,7 +71,9 @@
 "Form.validation.minLength: %@" = "Min length is %@";
 "Form.validation.maxLength: %@" = "Max length is %@";
 "Form.validation.exactLength: %@" = "Required length is %@";
-"Form.validation.pattern" = "Invalid value";
+"Form.validation.pattern.name" = "Only letters, spaces and hyphens are allowed";
+"Form.validation.pattern.city" = "Only letters, numbers, spaces and -,;()' are allowed";
+"Form.validation.pattern.street" = "Must contains letters and numbers";
```
Comment (bonkey) [inline AtlasUI/AtlasUI/en.lproj/Localizable.strings]: Please ask Raphael for translations, too.

Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/en.lproj/Localizable.strings]: I asked Raphael and he didn't like it and said that let's stick with the shop as they don't state the validations explicitly

**Comment by hani-ibrahim** [comment]
Won't be fixed

File: AtlasUI/AtlasUI/en.lproj/Localizable.strings (hunk 1 above)
Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/en.lproj/Localizable.strings]: fixed

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**